packages/core/src/pubsub/pubsub-message.ts

```diff
@@ -96,6 +97,7 @@ export function serialize(message: PubsubMessage): Uint8Array {
         doc: message.stream.toString(),
         stream: message.stream.toString(),
         tip: message.tip.toString(),
+        ...(message.model && { model: message.model.toString() }), // note: conditionally add the model key if message.model is not undefined        
       }
```

Comment:
```suggestion
      }
      if (message.model) payload.model = message.model.toString()
```
I think we can make this a little easier to read.